module/VuFind/tests/unit-tests/src/VuFindTest/Form/Handler/EmailTest.php

```diff
@@ -92,7 +150,7 @@ public function testDefaultEmailBehaviorWithConfig(): void
     protected function getHandler(array $config = []): Email
     {
         return new Email(
-            $this->createMock(\Laminas\View\Renderer\RendererInterface::class),
+            $this->createMock(\Laminas\View\Renderer\PhpRenderer::class),
```

Comment:
Looks like this change was necessary because `Email::handle()` makes a `$this->viewRenderer->partial()` call. But that is a potentially polymorphic call, because `$viewRenderer` is typed as a `RendererInterface`.

I would rather change the `partial()` call in `Email::handle()` to a `render()` call, because `render()` behaviour [has changed](https://openlibraryfoundation.atlassian.net/issues/VUFIND-1396) to match the old `partial()` variable scope functionality.

Reply:
Good catch -- thanks!